gitlab/utils.py

```diff
@@ -47,6 +49,39 @@ def response_content(
     return None
 
 
+def _transform_types(
+    data: Dict[str, Any], custom_types: dict, transform_files: Optional[bool] = True
```

Comment:
Forcing keyword arguments would improve readability, in my opinion.

```suggestion
    *, data: Dict[str, Any], custom_types: dict, transform_files: Optional[bool] = True
```

Reply:
I forced keyword arguments on the optionals after required args, to have some middle ground between clarity and brevity if that makes sense :)